lib/trento/domain/cluster/cluster.ex

```diff
@@ -233,7 +233,10 @@ defmodule Trento.Domain.Cluster do
   end
 
   def execute(%Cluster{cluster_id: nil}, _),
-    do: {:error, :cluster_not_found}
+    do: {:error, :cluster_not_registered}
```

Comment:
The same here.
There are some events (details updated, health changed...) on top of this.
We should exclude them.
I think we only need to leave the first 2 `execute`s (and later on the restoration part, which is to come)